app/main/forms.py

```diff
@@ -0,0 +1,22 @@
+from flask_wtf import FlaskForm
```

Comment:
We don't have any analytics cookies currently so I think we could delete related code if that doesn't break anything.

Reply:
Alright, I will remove.